apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/grpc/StargateBridgeInterceptor.java

```diff
@@ -35,18 +35,22 @@ public class StargateBridgeInterceptor implements ClientInterceptor {
   public <ReqT, RespT> ClientCall<ReqT, RespT> interceptCall(
       MethodDescriptor<ReqT, RespT> method, CallOptions callOptions, Channel next) {
 
+    // get default metadata
+    Metadata metadata = metadataResolver.getDefaultMetadata();
+
     // StargateRequestInfo is request scoped bean
     // there is no better way to know if if's available
     // as we can intercept outside request
     // thus ensure context is active and has instances
     InjectableContext.ContextState contextState =
         Arc.container().requestContext().getStateIfActive();
-    if (null == contextState || !hasStargateRequestBean(contextState.getContextualInstances())) {
-      return next.newCall(method, callOptions);
```

Comment:
So this call would not pass default metadata (for now mostly SourceApi) so need to change to pass it?

Reply:
Yup, at least sending the default metadata..